langDir/fr.json

```diff
@@ -112,25 +112,25 @@
-        "l1": "[Créer Token](@.common:links.rmrk_create.href)"
+        "m": "Tout d'abord, vous devez installer l'@.md:faq.a.2.l1, <b>sauvegardez votre graine</b>. Ensuite, vous rechargez l'adresse de votre portefeuille grace à notre fonction @.md:faq.a.1.l2 Une fois que vous avez les fonds, allez à @.md:faq.a.3.l1, créer Collection. Attendez que la collection soit émise, puis allez à @.md:faq.a.7.l1 et émettez votre tout premier jeton ! Si vous avez eu des problèmes, nous travaillons sur notre flux d'UX. Le code de KodaDot a environ un mois d'existance et il y a encore des possibilités d'amélioration, supportez-nous, nous ne sommes qu'une équipe de deux. Bonne frappe de la monnaie 😌",
+        "l1": "[Créer un jeton](@.common:links.rmrk_create.href)"
       },
       "8": {
         "m": "Nous explorons les possibilités, nous prévoyons en fait de rechercher activement à ce sujet dans notre étape 2"
       },
       "9": {
-        "m": "Nous essayons de fournir des mises à jour quotidiennes et des corrections le plus rapidement possible. Il se peut que nous publiions une nouvelle version et que votre navigateur ait mis en cache l'ancienne. Si vous êtes sous<b>Windows</b>, essayez de maintenir la touche <b>Ctrl enfoncée et d'appuyer sur F5</b>. <b>Pour les utilisateurs de Mac</b>, maintenez les touches<b>CMD et Shift enfoncées et appuyez sur R</b>. Si cela n'a pas aidé, DMs sur Twitter avec capture d'écran où est ouverte la console du développeur."
+        "m": "Nous essayons de fournir des mises à jour quotidiennes et des corrections le plus rapidement possible. Il se peut que nous publiions une nouvelle version et que votre navigateur ait mis en cache l'ancienne. Si vous êtes sous <b>Windows</b>, essayez de maintenir la touche <b>Ctrl enfoncée et d'appuyer sur F5</b>. <b>Pour les utilisateurs de Mac</b>, maintenez les touches <b>CMD et Shift enfoncées et appuyez sur R</b>. Si cela n'a pas aidé, DMs sur Twitter avec capture d'écran où est ouverte la console du développeur."
       },
       "10": {
         "m": "SEND l'enverra à quelqu'un, CONSUME le brûlera, LIST le mettra en vente pour que quelqu'un puisse vous l'acheter."
       },
       "11": {
         "m": "La meilleure façon de procéder est d'utiliser le @.md:faq.a.11.l1 et de rechercher votre adresse. @.md:faq.a.11.l2",
         "l1": "[Subscan Explorer](@.common:links.kusama_scanner.href)",
-        "l2": "[Vous pouvez trouver un exemple de la façon dont NFT frappe dans l'explorateur, lorsque vous regardez les extrinsèques.](@.common:links.kusama_scanner.extrinsic.href)"
+        "l2": "[Vous pouvez trouver un exemple de la façon dont l\'NFT est émis dans l'explorateur, lorsque vous regardez les extrinsèques.](@.common:links.kusama_scanner.extrinsic.href)"
```

Comment:
```suggestion
        "l2": "[Vous pouvez trouver un exemple de la façon dont un NFT est émis dans l'explorateur, lorsque vous regardez les extrinsèques.](@.common:links.kusama_scanner.extrinsic.href)"
```

Reply:
"un" equals "an" for undefined target while " le " is used when willing to point out to a specific target, the equivalent of "the" in English. I would even argue it should be le and not " l' " as I previously suggested. 
It's up to you to decide what's best.

Reply:
french grammar, you can't use `l'` here

Reply:
Indeed as well, in written language, better use the proper grammar use le instead of " l' ", which only works in verbal / slang expression.